bot.py

```diff
@@ -139,15 +144,30 @@ async def get_rank(ctx, name: str):
         )
         await ctx.send(file=file, embed=create_embed(embed_data))
         # pylint: disable=broad-except
-    except Exception as e_values:
-        print(e_values)
+    except ApiError as e_values:
```

Comment:
hmm I don't like this because you are just looking at `ApiError`, but I see why. Did you think of any other possible cases that we should think about?

Reply:
do you still need `as e_values`

Reply:
The reason why I used `ApiError` is because
1. I ignore the space when the user inputs a summoner name and receive it as one summoner name
eg. `!rank life is so hard d4022` (the user try to search two summoner name at a time) but I receive as `life is so hard d4022` and then send it API to find. Finally, API will send `ERROR` due to the `life is so hard d4022` summoner name is not found. 
-> if more than one user is searched or an invalid user name is entered, check it at once.

2. Although the user entered a valid summoner name, `ApiError` was used to prepare for other API errors.

I need `e_values` cuz to check whether the summoner name is not found or other API errors like `if e_values.response.status_code == 404:`

Reply:
Finally, I've understood using` Exception` is better than `ApiError`. I tried to use `Exception` to catch any other possible errors.